generator/whitelist.ts

```diff
@@ -132,6 +132,16 @@ const whitelist: WhitelistConfig[] = [
         basePath: 'datashare/resource-manager',
         namespace: 'Microsoft.DataShare',
     },
+    {
+        basePath: 'digitaltwins/resource-manager',
+        namespace: 'Microsoft.DigitalTwins',
+        resourceConfig: [
+            {
+                type: 'integrationResources',
+                scopes: ScopeType.Tenant | ScopeType.ManagementGroup | ScopeType.Subcription | ScopeType.ResourceGroup,
```

Comment:
Where did you get this list of scopes from? Is this definitely correct?

Reply:
We used to check the swagger definition to find scope list. May i ask how to get the scopes accurately?

Reply:
There often isn't a way to figure it out programatically (which is why autorest hasn't been able to). In this case, they've just declared the scope as `/{scope}/`, which is ambiguous: https://github.com/Azure/azure-rest-api-specs/blob/bbc1fea7971c12706bf5255a43d5f4674b010838/specification/digitaltwins/resource-manager/Microsoft.DigitalTwins/preview/2020-03-01-preview/digitaltwins.json#L515

The only examples I can find show it being used as an extension resource (`ScopeType.Extension`), but I think the safest thing to do is to check with the RP. @sup-s - since you committed the original swagger spec that this is being generated from, are you able to confirm what scopes the `integrationResources` resource type is valid at?

Reply:
Per discussion offline, the scope should be `ScopeType.Extension` according to the description in swagger: https://github.com/Azure/azure-rest-api-specs/blob/bbc1fea7971c12706bf5255a43d5f4674b010838/specification/digitaltwins/resource-manager/Microsoft.DigitalTwins/preview/2020-03-01-preview/digitaltwins.json#L1433-L1440
@hunmix let's change it to Extension for now.